Source/Core/Absy.cs

```diff
@@ -63,18 +63,15 @@ public Absy(IToken tok)
     // We uniquely number every AST node to make them
     // suitable for our implementation of functional maps.
     //
-    public int UniqueId
-    {
-      get { return this.uniqueId; }
-    }
+    public int UniqueId => this.uniqueId;
 
     private const int indent_size = 2;
```

Comment:
This is unrelated to your diff.  But the name of this constant does not follow the naming convention.  I want to raise awareness so contributors can start cleaning up code as we go along.

Reply:
Another comment on this: For a property getter like `UniqueId` that is backed by a private field (like `uniqueId`), I think you can just write `public int UniqueId { get; }` and drop the explicit declaration of the private field. Other than needing a property getting to implement a property getting in an `interface`, I've never understood the big fuss about having property getters--so, speaking for myself, I would just declare this as a field `public int UniqueId;` and be done with it.

Reply:
> This is unrelated to your diff. But the name of this constant does not follow the naming convention. I want to raise awareness so contributors can start cleaning up code as we go along.

Ideally we would let the build detect improper naming, possibly after a solution wide automated rename to let all names match the convention. I don't know how to set that up but I hope to look into it at some point.

> Another comment on this: For a property getter like UniqueId that is backed by a private field (like uniqueId), I think you can just write public int UniqueId { get; } and drop the explicit declaration of the private field.

This too might be something we can have the build check for. I'll leave this code for now since changing it isn't in scope of the PR.

Reply:
> Ideally we would let the build detect improper naming, possibly after a solution wide automated rename to let all names match the convention. I don't know how to set that up but I hope to look into it at some point.
>  
> This too might be something we can have the build check for.

@zvonimir set up StyleCop for Boogie.  He might know how to get this done.

Reply:
So this is the file to look at: https://github.com/boogie-org/boogie/blob/master/Source/StyleCop.ruleset
Here I disabled a bunch of checks that were causing tons of style checking errors to get generated. At that point I did not have time to fix all of those, and the plan was to merge basic checking, and get back to these and take care of them one by one.
Note that StyleCop is far from perfect, but is the best style checker for C# I managed to fine.
Anyhow, you want to look for something like this in that file:
```
<Rule Id="SA1310" Action="None" /> <!-- Field should not contain an underscore -->
```
Then you remove this line and see what happens. Then you do all the fixes.
I hope this makes sense.